docs/api-reference/next/image.md

```diff
@@ -118,78 +154,82 @@ const MyImage = (props) => {
-
-Second, the `sizes` property configures how `next/image` automatically generates an image source set. If no `sizes` value is present, a small source set is generated, suitable for a fixed-size image. If `sizes` is defined, a large source set is generated, suitable for a responsive image. If the `sizes` property includes sizes such as `50vw`, which represent a percentage of the viewport width, then the source set is trimmed to not include any values which are too small to ever be necessary.
+- First, the value of `sizes` is used by the browser to determine which size of the image to download, from an automatically-generated source set by `next/image`. When the browser chooses, it does not yet know the size of the image on the page, so it selects an image that is the same size or larger than the viewport. The `sizes` property allows you to tell the browser that the image will actually be smaller than full screen. If you don't specify a `sizes` value in an image with the `fill` property, a default value of `100vw` (full screen width) is used.
+- Second, the `sizes` property configures how `next/image` automatically generates an image source set. If no `sizes` value is present, a small source set is generated, suitable for a fixed-size image. If `sizes` is defined, a large source set is generated, suitable for a responsive image. If the `sizes` property includes sizes such as `50vw`, which represent a percentage of the viewport width, then the source set is trimmed to not include any values which are too small to ever be necessary.
 
 For example, if you know your styling will cause an image to be full-width on mobile devices, in a 2-column layout on tablets, and a 3-column layout on desktop displays, you should include a sizes property such as the following:
 
-```js
+```jsx
 import Image from 'next/image'
-const Example = () => (
-  <div className="grid-element">
+
+export default function Page() {
+  ;<div className="grid-element">
     <Image
       src="/example.png"
-      fill
-      sizes="(max-width: 768px) 100vw,
-              (max-width: 1200px) 50vw,
-              33vw"
+      layout="fill"
+      sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
     />
   </div>
-)
+}
 ```
 
 This example `sizes` could have a dramatic effect on performance metrics. Without the `33vw` sizes, the image selected from the server would be 3 times as wide as it needs to be. Because file size is proportional to the square of the width, without `sizes` the user would download an image that's 9 times larger than necessary.
 
-Learn more about `srcset` and `sizes`:
+Learn more about [`srcset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-srcset) and [`sizes`](https://web.dev/learn/design/responsive-images/#sizes).
```

Comment:
Lets change these links back to how they were. Linking to `srcset` is useless because the user cant set that prop, we autogenerate it.

Reply:
I prefer having this more streamlined, rather than text that says web.dev and mdn

Reply:
@leerob Its not about the order, its about the actual links. 

Linking to `srcset` is not as helpful, we need to link to `sizes`.